include/snowflake/client.h

```diff
@@ -434,6 +434,16 @@ typedef struct SF_CHUNK_DOWNLOADER SF_CHUNK_DOWNLOADER;
  */
 typedef struct SF_PUT_GET_RESPONSE SF_PUT_GET_RESPONSE;
 
+typedef void* result_set_ptr;
+
+/**
+ * An enumeration over all supported query result formats.
+ */
+typedef enum QueryResultFormat
+{
+  SF_ARROW_FORMAT, SF_JSON_FORMAT, SF_FORMAT_MAX
+} QueryResultFormat_t;
```

Comment:
cannot we name the type the same as the enum? without _t?

Reply:
done